packages/react-ai/src/components/AIConversation/views/default/__tests__/Form.spec.tsx

```diff
@@ -57,4 +58,57 @@ describe('Form', () => {
     expect(fileInput.files).not.toBeNull();
     expect(fileInput.files![0]).toStrictEqual(testFile);
   });
+
+  it('updates input value through multiple change events with IME characters', async () => {
+    const result = render(<Form {...defaultProps} />);
+    expect(result.container).toBeDefined();
+
+    const textInput: HTMLInputElement = screen.getByTestId('text-input');
+
+    const preMatureInput = Object.defineProperty(textInput, 'value', {
+      configurable: true,
+      value: '你',
+    });
+
+    const completeInput = Object.defineProperty(textInput, 'value', {
+      configurable: true,
+      value: '你好',
+    });
+
+    await waitFor(() => fireEvent.change(textInput, preMatureInput));
+
+    await waitFor(() => fireEvent.change(textInput, completeInput));
+
+    expect(textInput.value).not.toBeNull();
+    expect(textInput.value).toBe('你好');
+  });
+
+  it('updates input value correctly after IME composition successfully ends', async () => {
+    const result = render(<Form {...defaultProps} />);
+    expect(result.container).toBeDefined();
+
+    const textInput: HTMLInputElement = screen.getByTestId('text-input');
+
+    const preMatureInput = Object.defineProperty(textInput, 'value', {
+      configurable: true,
+      value: 'しあわせ',
+    });
+
+    const completeInput = Object.defineProperty(textInput, 'value', {
+      configurable: true,
+      value: '幸せならおkです',
+    });
+
+    await waitFor(() => {
+      fireEvent.compositionStart(textInput);
+      fireEvent.change(textInput, preMatureInput);
+    });
+
+    await waitFor(() => {
+      fireEvent.compositionEnd(textInput, completeInput);
+    });
+
+    expect(textInput.value).not.toBeNull();
```

Comment:
When does `textInput.value` resolve to `null`?

Reply:
Ah realized it can only be an empty value and can't resolve to `null`.